libobs/audio-monitoring/osx/coreaudio-output.c

```diff
@@ -288,6 +296,8 @@ static void audio_monitor_init_final(struct audio_monitor *monitor)
 
 	obs_source_add_audio_capture_callback(monitor->source,
 					      on_audio_playback, monitor);
+	obs_source_add_audio_pause_callback(monitor->source, on_pause_audio,
```

Comment:
The callback is added but never removed (even though the corresponding function is added) - is that on purpose?

Reply:
That was an overlook. Added removing the callback in `static void audio_monitor_free()` function.